config/google_cloud_exporter/iis/config.yaml

```diff
@@ -0,0 +1,30 @@
+receivers:
+  iis:
+    collection_interval: 10s
+
+processors:
```

Comment:
We probably need resource detection, resource, and resource attributes transposer here. I am curious to see how the metrics look into the metric explorer.

Reply:
See Brandon's comment